src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshService.java

```diff
@@ -117,4 +133,7 @@ private void processSingleMessage(String messageId) {
         }
     }
 
+    private boolean isPathologyOrScreeningMessage(WorkflowId workflowId) {
+        return workflowId.equals(WorkflowId.PATHOLOGY) || workflowId.equals(WorkflowId.SCREENING);
+    }
```

Comment:
Might suggest making this more general in nomenclature. `isSupportedMessage()`, perhaps. The set of supported workflow IDs can then be an extracted piece of class-level knowledge. If the supported workflows would ever change then we'd only have to change the members of the set, not the name of the method too.
```
private static final Set<WorkflowId> SUPPORTED_WORKFLOWS = Set.of(WorkflowId.PATHOLOGY, WorkflowId.SCREENING);
```
```suggestion
    private boolean isSupportedMessage(WorkflowId workflowId) {
        return SUPPORTED_WORKFLOWS.contains(workflowId);
    }
```
I appreciate that the set of supported workflows is very unlikely to change, so might be unnecessary.